### DISCUSSION TITLE
Airgapped Sheihak

### DISCUSSION THREAD

**Discussion by datgous** [witnet/sheikah#1757] [Ideas]
Dear Sheikah devs

I value in a wallet the ability to do **qr-based transaction signing** from an old, airplane-mode mobile (à la [parity-signer](https://www.parity.io/technologies/signer/), or [airgap.it](https://airgap.it/)), as it offers that extra level of isolation that might just have your back if(when) your computer is compromised.

I realise Sheikah does not have a mobile version (yet?), so I am just tossing the idea for you to consider in case you ever go down that path.

If the development effort of starting such a project from scratch is too high, I have seen in airgap's telegram channel that they are open to collaborations, more so with open source projects. So that could also be an opportunity to explore. 

Thank you, keep the great work! 🤟

**Comment by aesedepece** [comment]
This is very interesting @datgous. A light mobile Witnet wallet is in the making and will be released soon indeed. Air gapped transaction signature would be a nice feature addition for sure :thinking:
